pom.xml

```diff
@@ -465,6 +469,8 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- Disable this plugin after 12.3.0 upgrade -->
```

Comment:
Why?

Reply:
I had some issues during the upgrade, specially while dealing with the custom jackrabbit version. I think you also had the same issues when this enforcer plugin has been introduced, but what you probably forced specific version of the dependencies directly in the project root pom file (this one). 

In my opinion the solution you adopted to make this maven plugin happy is not really a solution, because you may override dependency versions defined in the product pom(s). 

You need to consider that our product is tested and shipped with specific artifact versions and overriding those dependencies in the project is not really an option. I can understand if those dependencies are project related, but in case they are used by the product they shouldn't be overwritten. 

IMHO this enforcer plugin is making the project not easy to maintain/upgrade and there isn't really a cost/benefit.

Reply:
Are you saying that the versions enforced by 'our' POM were actually different from those specified by the parent (Hippo) POM, @giacomolm? I'd think they should be the same, meaning that on upgrade, they should simply be upgraded in line with the parent POM. If they were different, this was probably not intentional.

Reply:
What's your solution to fix the fact that some dependencies need higher versions of other dependencies? Without this plugin we could get method not found exceptions etc. when using the other dependency

Reply:
@giacomolm hi - please can you comment here re possible alternative solutions for tackling dependency problems.  As I understand it the enforcer plug-in helps identifies these at compile-time, meaning they can be discussed and resolved, hence mitigating risk of them only be found at run-time.

Reply:
@giacomolm @jacsamell As discussed, we've decided (I'm happy to own decision) to go ahead with use of enforcer for now with v12.3.0, i.e. keep manually updating dependencies identified by enforcer, in CMS pom.xml.  Thereafter, see what BR Engineering suggest for 12.4.0 onwards.